### ISSUE TITLE
<add> output font size

### ISSUE THREAD

**Issue by wehooper** [IUBLibTech/newton_chymistry#42]
In \<add\> tags, we have been reducing the font size of the added text to the same font size used in \<hi\> elements, which I can't remember now, apologies.

A good example in Portsmouth 3973, page 1, line beginning "colour of ♁", which ends with an \<add place="inline"\> then another \<add place="infralinear"\>.

Compare font sizes:
P4: http://carbon.dlib.indiana.edu:8215/newton-dev/mss/norm/ALCH00109
P5: http://carbon.dlib.indiana.edu:8220/text/ALCH00109/diplomatic#?c=&m=&s=&cv=

**Comment by Conal-Tuohy** [comment]
@wehooper
Comparing the font sizes I saw that the two pages referred to above have different font sizes because one is a link to a diplomatic page, and the other is normalized. It was confusing because the new page showed a reduced font size, whereas in the old page the entire line used a single font size. I have updated the TEI CSS now, so the result will be different from when you posted the issue, but I've checked this MS on both sites, in diplomatic and normalized views, and they appear to match, so I think this is sorted.
